autoload/asyncomplete/sources/flow.vim

```diff
@@ -66,6 +70,21 @@ function! asyncomplete#sources#flow#get_source_options(opts)
        \ })
 endfunction
 
+let s:flowbin_path_by_dir = {} " dir: <path to flow>
+
+function! s:resolve_flowbin_path(file, fallback)
+    let l:dir = fnamemodify(a:file, ':h')
+    if !has_key(s:flowbin_path_by_dir, l:dir)
+        let l:node_dir = asyncomplete#utils#find_nearest_directory(a:file, 'node_modules')
+        if filereadable(l:node_dir . '/.bin/flow')
```

Comment:
for windows it would be `./bin/flow.cmd`. Let me know if you don't have windows and I can fix it.